community-containers/memories/memories.json

```diff
@@ -0,0 +1,36 @@
+{
+    "aio_services_v1": [
+        {
+            "container_name": "nextcloud-aio-memories",
+            "display_name": "Transcoding container for memories",
+            "documentation": "https://github.com/nextcloud/all-in-one/tree/main/community-containers/memories",
+            "image": "radialapps/go-vod",
+            "image_tag": "latest",
+            "internal_port": "47788",
+            "restart": "unless-stopped",
+            "environment": [
+                "TZ=%TIMEZONE%",
+                "NEXTCLOUD_HOST=https://%NC_DOMAIN%"
+            ],
+            "volumes": [
+                {
+                    "source": "%NEXTCLOUD_DATADIR%",
+                    "destination": "/mnt/ncdata",
+                    "writeable": false
+                }
+            ],
+            "devices": [
+                "/dev/dri"
+            ],
+            "aio_variables": [
+                "nextcloud_enable_dri_device=true"
```

Comment:
I wonder is the container even useful if no /dev/dri device is present on the host?

Reply:
Actually it is. The ffmpeg image is guaranteed to have a newer version of ffmpeg giving better format support etc.

Reply:
All right. Then I guess remocing this option makes most sense.

Reply:
Done via https://github.com/nextcloud/all-in-one/pull/3736#discussion_r1391778560